symlink-vendor-directory.js

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable flowtype/require-valid-file-annotation */
+/* eslint-disable import/no-nodejs-modules */
+
+const {exec} = require('child_process'); //
+const path = require('path');
+
+const parameters = process.argv.slice(2);
+
+const from = parameters.length > 1 ? parameters[0] : '../../vendor';
+const to = parameters.length > 2 ? parameters[1] : 'node_modules/@sulu/vendor';
+const ignoreCheck = parameters.length > 3 && parameters[2] === 'ignoreCheck';
```

Comment:
think i would call this `--force` if we want to make this configurable 🙂 
```suggestion
const doNotCheckCurrentDirectory = parameters.length > 3 && parameters[2] === '--force';
```